AddOns/BasicFinancials/app/src/codeunit/AssistedSetupBF.Codeunit.al

```diff
@@ -0,0 +1,46 @@
+codeunit 20605 "Assisted Setup BF"
+{
+    Access = Internal;
+
+    var
+        AssistedSetupTxt: Label 'Set up the Business Central Basic extension';
+        AssistedSetupHelpTxt: Label 'https://go.microsoft.com/fwlink/?linkid=', Locked = true;
+        AssistedSetupDescriptionTxt: Label 'Business Central Basic';
```

Comment:
How about:

Start using the Basic Experience extension to manage your business.